From 297f798470e89b131d67127d734b749fab5111e3 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Wed, 31 Jul 2024 13:56:59 +0300
Subject: [PATCH 01/18] arm64: renesas: solidrun: put /dts-v1/ at start of dts
 only

The device-tree language version indicator should be placed at top dts
only, not in includes.
Update all solidrun dts to specify it as the first statement
consistently.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm64/boot/dts/renesas/rzg2l-hummingboard-pro.dts    | 3 ++-
 arch/arm64/boot/dts/renesas/rzg2l-hummingboard-ripple.dts | 3 ++-
 arch/arm64/boot/dts/renesas/rzg2l-hummingboard-tutus.dts  | 3 ++-
 arch/arm64/boot/dts/renesas/rzg2l-sr-som.dtsi             | 2 --
 arch/arm64/boot/dts/renesas/rzg2lc-hummingboard.dts       | 3 ++-
 arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi            | 1 -
 arch/arm64/boot/dts/renesas/rzv2l-hummingboard-pro.dts    | 3 ++-
 arch/arm64/boot/dts/renesas/rzv2l-hummingboard-ripple.dts | 3 ++-
 arch/arm64/boot/dts/renesas/rzv2l-sr-som.dtsi             | 2 --
 9 files changed, 12 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-pro.dts b/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-pro.dts
index df420c51d..390acbfcd 100644
--- a/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-pro.dts
+++ b/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-pro.dts
@@ -5,6 +5,8 @@
  * Copyright (C) 2023 SolidRun Ltd.
  */
 
+/dts-v1/;
+
 /*
  * DIP-Switch S3 setting on SoM
  * 1 : High; 0: Low
@@ -13,7 +15,6 @@
 
 #define SW_SD0_DEV_SEL	1
 
-/dts-v1/;
 #include "rzg2l-sr-som.dtsi"
 #include "rz-hummingboard-common.dtsi"
 
diff --git a/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-ripple.dts b/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-ripple.dts
index d410062dc..43dce28f9 100644
--- a/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-ripple.dts
+++ b/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-ripple.dts
@@ -5,6 +5,8 @@
  * Copyright (C) 2023 SolidRun Ltd.
  */
 
+/dts-v1/;
+
 /*
  * DIP-Switch S3 setting on SoM
  * 1 : High; 0: Low
@@ -13,7 +15,6 @@
 
 #define SW_SD0_DEV_SEL	1
 
-/dts-v1/;
 #include "rzg2l-sr-som.dtsi"
 #include "rz-hummingboard-common.dtsi"
 
diff --git a/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-tutus.dts b/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-tutus.dts
index 42bc0ec7c..3730383ff 100644
--- a/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-tutus.dts
+++ b/arch/arm64/boot/dts/renesas/rzg2l-hummingboard-tutus.dts
@@ -5,6 +5,8 @@
  * Copyright (C) 2021 Renesas Electronics Corp.
  */
 
+/dts-v1/;
+
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/pinctrl/rzg2l-pinctrl.h>
 #include <dt-bindings/input/input.h>
@@ -17,7 +19,6 @@
 
 #define SW_SD0_DEV_SEL	1
 
-/dts-v1/;
 #include "rzg2l-sr-som.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/renesas/rzg2l-sr-som.dtsi b/arch/arm64/boot/dts/renesas/rzg2l-sr-som.dtsi
index 9b74f6582..9e2c412c7 100644
--- a/arch/arm64/boot/dts/renesas/rzg2l-sr-som.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzg2l-sr-som.dtsi
@@ -5,8 +5,6 @@
  * Copyright (C) 2023 SolidRun Ltd.
  */
 
-/dts-v1/;
-
 /* RZ/G2L R9A07G044L2 SoC */
 #include "r9a07g044l2.dtsi"
 #include "rz_2l-sr-som-common.dtsi"
diff --git a/arch/arm64/boot/dts/renesas/rzg2lc-hummingboard.dts b/arch/arm64/boot/dts/renesas/rzg2lc-hummingboard.dts
index 931a53705..2dd05a086 100644
--- a/arch/arm64/boot/dts/renesas/rzg2lc-hummingboard.dts
+++ b/arch/arm64/boot/dts/renesas/rzg2lc-hummingboard.dts
@@ -5,6 +5,8 @@
  * Copyright (C) 2021 Renesas Electronics Corp.
  */
 
+/dts-v1/;
+
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/pinctrl/rzg2l-pinctrl.h>
 
@@ -16,7 +18,6 @@
 
 #define SW_SD0_DEV_SEL	1
 
-/dts-v1/;
 #include "rzg2lc-sr-som.dtsi"
 #include "rz-hummingboard-common.dtsi"
 
diff --git a/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi b/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi
index 22251da97..15251ae1f 100644
--- a/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzg2lc-sr-som.dtsi
@@ -5,7 +5,6 @@
  * Copyright (C) 2021 Renesas Electronics Corp.
  */
 
-/dts-v1/;
 /* RZ/G2LC R9A07G044C2 SoC */
 #include "r9a07g044c2.dtsi"
 
diff --git a/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-pro.dts b/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-pro.dts
index 061b8851c..1690b4235 100644
--- a/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-pro.dts
+++ b/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-pro.dts
@@ -5,6 +5,8 @@
  * Copyright (C) 2023 SolidRun Ltd.
  */
 
+/dts-v1/;
+
 /*
  * DIP-Switch S3 setting on SoM
  * 1 : High; 0: Low
@@ -13,7 +15,6 @@
 
 #define SW_SD0_DEV_SEL	1
 
-/dts-v1/;
 #include "rzv2l-sr-som.dtsi"
 #include "rz-hummingboard-common.dtsi"
 
diff --git a/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-ripple.dts b/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-ripple.dts
index c60563e9a..7f817849b 100644
--- a/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-ripple.dts
+++ b/arch/arm64/boot/dts/renesas/rzv2l-hummingboard-ripple.dts
@@ -5,6 +5,8 @@
  * Copyright (C) 2023 SolidRun Ltd.
  */
 
+/dts-v1/;
+
 /*
  * DIP-Switch S3 setting on SoM
  * 1 : High; 0: Low
@@ -13,7 +15,6 @@
 
 #define SW_SD0_DEV_SEL	1
 
-/dts-v1/;
 #include "rzv2l-sr-som.dtsi"
 #include "rz-hummingboard-common.dtsi"
 
diff --git a/arch/arm64/boot/dts/renesas/rzv2l-sr-som.dtsi b/arch/arm64/boot/dts/renesas/rzv2l-sr-som.dtsi
index 3ab7ab120..8886c48f1 100644
--- a/arch/arm64/boot/dts/renesas/rzv2l-sr-som.dtsi
+++ b/arch/arm64/boot/dts/renesas/rzv2l-sr-som.dtsi
@@ -5,8 +5,6 @@
  * Copyright (C) 2023 SolidRun Ltd.
  */
 
-/dts-v1/;
-
 /* RZ/V2L R9A07G054L2 SoC */
 #include "r9a07g054l2.dtsi"
 #include "rz_2l-sr-som-common.dtsi"
-- 
2.43.0

